Vendor.Application/DTOs/VendorDTO/VendorByIdDto.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Vendor.Application.DTOs.VendorDTO
+{
+    public class VendorByIdDto
```

Comment:
Since createDto and VendorByIdDto have same fields move the common fields to another  dto class ex vendordto and inherit vendordto in vendorByIdDto

Reply:
done